.github/ISSUE_TEMPLATE/request.yml

```diff
@@ -2,7 +2,7 @@
 name: Service Request
 description: A service request to the MIRSG team
 title: "📥 [REQUEST]: "
-labels: ["request"]
+labels: ["request"] # yamllint disable-line rule:quoted-strings rule:brackets
```

Comment:
the `brackets` will not need to be ignored once this PR is merged, and the version of the hooks used in this repo is updated to 0.49 (as changes in this PR mean that only non-empty brackets are forbidden)